### ISSUE TITLE
Last seen date

### ISSUE THREAD

**Issue by martinig** [KluaneRedSquirrelProject/KRSP-Database-Errors#93]
squirrel 5682 has a last seen date of 2003-06-08 in the flastall2 LTD, but it was recorded as the focal in a mating chase in Jeff's mating chase data where it was last seen 2004-04-11 for a few hours while he followed it around.

**Comment by agmcadam** [comment]
There is no record of 5682 in the behaviour table.  Perhaps Jeff's focal mating chase data are not entered in the behavior table.

**Comment by martinig** [comment]
@agmcadam Would it be helpful if I sent you the mating chase data?

**Comment by agmcadam** [comment]
Yes, but it will not get in this year.  It would need to wait until next year.

**Comment by martinig** [comment]
If possible, please leave this open for me in that case:

- [ ] send mating chase data to @agmcadam
